hack/arktos-cni.rc

```diff
@@ -134,32 +149,50 @@ ensure_devstack() {
-        sudo mv ${containerd_path} ${containerd_back_path}
-        sudo cp ${CONTAINERD_DIR}/bin/containerd ${containerd_path}
-        sudo systemctl start containerd
-        echo "warning: original containerd binary is saved to ${containerd_path})/containerd_arktos_backup"
-        echo "...done"
+#    local CONTAINERD_DIR=$(dirname "$BASH_SOURCE[0]}")/../../arktos-containerd
+#    [ -d ${CONTAINERD_DIR} ] || die "arktos-containerd repo not found. please clone futurewei-cloud/arktos-containerd branch wip-pod-annotation-to-cni-args side by side with arktos"
+#    if [ ! -x ${CONTAINERD_DIR}/bin/containerd ]; then
+#        echo "building arktos containerd..."
+#        pushd ${CONTAINERD_DIR} > /dev/null
+#        docker run --rm -e GOPATH=/go -v ${PWD}:/go/src/github.com/containerd/containerd -w /go/src/github.com/containerd/containerd golang:1.12 sh -c "apt update; apt install -y libbtrfs-dev libseccomp-dev; make bin/containerd"
+#        popd > /dev/null
+#        echo "...built"
+#    fi
+#
+#    echo "make sure arktos containerd in power"
+#    if [ ! "$(${CONTAINERD_DIR}/bin/containerd --version)" == "$(containerd --version)" ]; then
+#        echo "stopping containerd shortly to replace the binary..."
+#        local containerd_path=$(which containerd)
+#        sudo systemctl stop containerd
+#        local containerd_back_path=$(dirname ${containerd_path})/containerd_arktos_backup
+#        sudo mv ${containerd_path} ${containerd_back_path}
+#        sudo cp ${CONTAINERD_DIR}/bin/containerd ${containerd_path}
+#        sudo systemctl start containerd
+#        echo "warning: original containerd binary is saved to ${containerd_path})/containerd_arktos_backup"
+#        echo "...done"
+#    fi
+
+    [ ! -d "$HOME/arktos-containerd" ] && mkdir $HOME/arktos-containerd
+    local CONTAINERD_DIR=$HOME/arktos-containerd
+    [ ! -f "$CONTAINERD_DIR/containerd" ] && wget -qO- https://github.com/CentaurusInfra/containerd/releases/download/tenant-cni-args/containerd.zip | zcat > $CONTAINERD_DIR/containerd && sudo chmod +x $CONTAINERD_DIR/containerd
+    echo "Checking arktos containerd..."
+    if [ ! "$(${CONTAINERD_DIR}/containerd --version)" == "$(containerd --version)" ] ; then
+      echo "arktos containerd not found..."
+      [ ! -f "$HOME/arktos-containerd/containerd-1.4.2-linux-amd64.tar.gz" ] && wget https://github.com/containerd/containerd/releases/download/v1.4.2/containerd-1.4.2-linux-amd64.tar.gz -P $HOME/arktos-containerd/
+      tar -zxf $HOME/arktos-containerd/containerd-1.4.2-linux-amd64.tar.gz --directory /usr/
+      echo "installing arktos containerd..."
+      sudo systemctl stop containerd
+      sudo mv /usr/bin/containerd /usr/bin/containerd.bak
+      sudo cp $HOME/arktos-containerd/containerd /usr/bin/
+      sudo systemctl restart containerd
+      sudo systemctl restart docker
+    else
+      echo "arktos containerd found..."
     fi
 }
 
 
-CNIPLUGIN=${CNIPLUGIN:-"bridge"}
+CNIPLUGIN=${CNIPLUGIN:-"mizar"}
```

Comment:
we should not change the default from bridge to mizar. 
If we want to deploy mizar, we should be able to do
```
CNIPLUGIN=mizar ./hack/arktos-up.sh
```

Reply:
Okay, we will change the code as per your suggestions.
We'll use CNIPLUGIN environment variable to set mizar as default CNI while deploying arktos.